src/framework/audio/internal/synthesizers/fluidsynth/soundmapping.h

```diff
@@ -997,7 +997,9 @@ inline const ArticulationMapping& articulationSounds(const mpe::PlaybackSetupDat
     }
 
     if (setupData.category == mpe::SoundCategory::Winds) {
-        return WINDS;
+        if (setupData.id == mpe::SoundId::Horn || setupData.id >= mpe::SoundId::Tuba) {
```

Comment:
slight kludge, but at least it prevents all known woodwind instruments from using "muted trumpet" sound when marked as muted.